.eslintrc-webapp.json

```diff
@@ -66,24 +59,142 @@
           "component"
         ]
       }
-    ]
+    ],
+    "react/no-string-refs": 2,
+    "no-only-tests/no-only-tests": ["error", {"focus": ["only", "skip"]}],
+    "react/style-prop-object": [2, {
+      "allow": ["Timestamp"]
+    }]
   },
   "overrides": [
     {
-      "files": ["tests/**"],
+      "files": ["**/*.tsx", "**/*.ts"],
+      "extends": [
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "rules": {
+        "camelcase": 0,
+        "no-shadow": 0,
+        "import/no-unresolved": 0, // ts handles this better
```

Comment:
Weird that ESLint is fine with these comments since JSON doesn't technically allow them. I guess it doesn't actually read JSON since I don't think the other repos have a file extension on these files